components/TextControls/tests/unit/MDCBaseTextFieldTests.m

```diff
@@ -338,4 +338,108 @@ - (void)testPlaceholderVisibility {
                                      labelState:MDCTextControlLabelStateFloating]);
 }
 
+- (void)testDefaultAccessibilityLabelWithOnlyLabelText {
+  // Given
+  CGRect textFieldFrame = CGRectMake(0, 0, 130, 100);
+  MDCBaseTextField *textField = [[MDCBaseTextField alloc] initWithFrame:textFieldFrame];
+  NSString *labelText = @"label text";
+
+  // When
+  textField.label.text = labelText;
```

Comment:
Yes and let us test the attributedText too!

https://developer.apple.com/documentation/uikit/uitextfield/1619602-attributedtext?language=objc

Reply:
Hmmm I don't want people to use the attributedText property on the label. Remember how I mentioned that I was considering making the label private and only exposing a pass through property `labelText`? This is why :/